litmus/director/create-apikey/test.yaml

```diff
@@ -45,6 +46,30 @@
             username: "{{ credentials.stdout_lines[0] }}"
             password: "{{ credentials.stdout_lines[1] }}"
 
+        - name: Fetch user details
+          uri:
+            url: "{{ director_url }}/v3/accounts/{{ accountId }}"
+            url_username: "{{username}}"
+            url_password: "{{password}}"
+            method: GET
+            status_code: 200
+          register: user_details
+
+        - set_fact: 
+            user_email: "{{user_details.json.email}}"
+
+        - name : Change value of onboardingStatus
+          uri:
+            url: "{{ director_url }}/v3/accounts/{{ accountId }}"
+            url_username: "{{username}}"
+            url_password: "{{password}}"
+            method: PUT
+            body_format: json
+            body:
+              {"firstName":"{{ firstname }}", "lastName":"{{ surname }}","email":"{{ user_email }}", "onboardingStatus":3}
```

Comment:
Can this `onboardingStatus` number `3` be set dynamically just like the `{{username}}`? If this involves a non-trivial effort or requires you to more than 2 minutes please skip this review comment.

Reason:
The DirectorOnline team doesn't know the future UI workflow as of now, it shouldn't change in this sprint for sure, but it might end up changing in 1-3 months.